client/flutter/lib/components/home_page_header.dart

```diff
@@ -0,0 +1,187 @@
+                            color: CupertinoColors.white,
+                            fontWeight: FontWeight.w600,
+                          ),
+                        ),
+                      ),
+                      Container(
+                        margin: EdgeInsets.symmetric(
+                          horizontal: 14,
+                        ),
+                        color: Color.fromARGB(127, 250, 232, 169),
+                        width: 1,
+                      ),
+                      Text(
+                        "COVID-19 Response",
+                        style: TextStyle(
+                          color: Color(0xffFAE8A9),
+                          fontWeight: FontWeight.w600,
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                SizedBox(
+                  height: 32,
+                ),
+                Text(
+                  this.title(context),
+                  style: TextStyle(
+                    color: CupertinoColors.white,
+                    fontSize: 48,
+                    fontWeight: FontWeight.w900,
+                  ),
+                  textAlign: TextAlign.left,
+                ),
+                Text(
+                  "See what your symptoms could mean and what you can do to move forward.",
+                  style: TextStyle(
+                    color: CupertinoColors.white,
+                    fontSize: 16,
+                    height: 1.37,
+                  ),
+                ),
+                Container(
+                  height: 24,
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    SvgPicture.asset(this.svgAssetName),
+                    Padding(
+                      padding: EdgeInsets.symmetric(
+                        horizontal: 12,
+                      ),
+                      child: CupertinoButton(
+                        borderRadius: BorderRadius.circular(50),
+                        padding: EdgeInsets.symmetric(
+                          horizontal: 32,
+                          vertical: 8,
+                        ),
+                        color: CupertinoColors.white,
+                        child: Text(
+                          this.buttonText(context),
+                          style: TextStyle(color: Constants.primaryColor),
+                        ),
+                        onPressed: () {},
+                      ),
+                    ),
+                  ],
+                )
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class HeaderClipper extends CustomClipper<Path> {
+  @override
+  Path getClip(Size size) {
+    Path path = Path();
+
+    path.lineTo(0, size.height);
+    path.arcToPoint(Offset(size.width, size.height - 20),
+        radius: Radius.elliptical(size.width, 240));
```

Comment:
should 240 scale with screen size

Reply:
It could however it’s worrying that on screens like iPads it would be huge.

Reply:
You're right let's leave it alone for now.

It might change later so let's not mess with it for now.